apps/evm/src/clients/api/queries/useGetPools/__tests__/__snapshots__/index.spec.ts.snap

```diff
@@ -1090,7 +1090,7 @@ exports[`useGetPools > returns pools with time based reward rates in the correct
           "liquidityCents": "1.000006200805040439464388948970138750743214608e+23",
           "reserveFactor": 1,
           "reserveTokens": "0",
-          "supplierCount": 10,
+          "supplierCount": 0,
```

Comment:
It looks like this isn't getting mocked

Reply:
After reviewing this, I'd actually say this is right.

Or maybe not right per se, but it's more precise: this test mocks the chain to be Arbitrum, becase we want to test time based reward rates. But the other depending mocks are all BSC based: tokens, pools, participant counts, etc. I'd say that all those would have to be mocked differently so that the resulting snapshot makes better sense.

Even better would be not snapshotting the whole pools array, just the time based reward rates. I feel both are kind of out of scope from this PR, but let me know if I should address this here or in a new one.

Reply:
Ok I see, that makes sense.
I'm ok with leaving the PR like this for now, I agree it's out of scope.